src/app/components/button/button.ts

```diff
@@ -66,7 +69,7 @@ export class ButtonDirective implements AfterViewInit, OnDestroy {
 
     setStyleClass() {
         let styleClass = this.getStyleClass();
-        this.el.nativeElement.className = styleClass;
+        this.el.nativeElement.className = styleClass + this._initialStyleClass;
```

Comment:
It should be:
`this.el.nativeElement.className` = styleClass + ' ' + this._initialStyleClass;

Reply:
yes i fix this in https://github.com/primefaces/primeng/pull/9041